src/commands/deploy/index.ts

```diff
@@ -100,11 +100,13 @@ Support for package.json's main field, and intrinsic index.js entrypoints are co
       '-b, --branch <name>',
       'Serves the same functionality as --alias. Deprecated and will be removed in future versions',
     )
-    .option('-o, --open', 'Open site after deploy', false)
+    .option('-O, --open', 'Open site after deploy', false)
```

Comment:
Same here, there's already `-o` for `--offline` in the BaseCommand, so `deploy -o` wasn't working as intended. It just started an offline deploy which of course didn't work.